client/src/components/auth/VerifiedEmailComponent.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <div></div>
+</template>
+
+<script>
+import * as actions from '@/store/modules/auth/types/actions';
+import { mapActions } from 'vuex';
+export default {
+    name: 'VerifiedEmailComponent',
+
+    methods: {
+        ...mapActions('auth', {
+            verifyEmail: actions.VERIFY_EMAIL
+        })
+    },
+
+    async created() {
+        await this.verifyEmail(this.$route.query);
+        this.$router.push({ name: 'SignIn' });
```

Comment:
Instead of redirect, display success message and link to log in page, as I wrote above